### ISSUE TITLE
Jupyter Kite not working

### ISSUE THREAD

**Issue by swankyshahir** [kiteco/issue-tracker#336]
OS _ Windows 10
Kite desktop app installed - Yes
The error you received when trying to install the plugin from the command line :  it is getting installed but not detected by KITE or Jupyter notebook
The output of pip list : 
Package                            Version
---------------------------------- -------------------
-umpy                              1.18.1
absl-py                            0.9.0
alabaster                          0.7.12
anaconda-client                    1.7.2
anaconda-navigator                 1.9.12
anaconda-project                   0.8.3
argh                               0.26.2
argon2-cffi                        20.1.0
asn1crypto                         1.4.0
astor                              0.8.0
astroid                            2.4.2
astropy                            4.0.1.post1
atomicwrites                       1.4.0
attrs                              19.3.0
autopep8                           1.4.4
autoviml                           0.1.621
autoviz                            0.0.68
Babel                              2.8.0
backcall                           0.2.0
backports.functools-lru-cache      1.6.1
backports.shutil-get-terminal-size 1.0.0
backports.tempfile                 1.0
backports.weakref                  1.0.post1
bcrypt                             3.1.7
beautifulsoup4                     4.9.1
bitarray                           1.4.2
bkcharts                           0.2
bleach                             3.1.5
blinker                            1.4
bokeh                              2.1.1
boto                               2.49.0
boto3                              1.13.3
botocore                           1.16.3
Bottleneck                         1.3.2
brotlipy                           0.7.0
cachetools                         4.1.1
catboost                           0.23
certifi                            2020.6.20
cffi                               1.14.0
chardet                            3.0.4
click                              7.1.2
cloudpickle                        1.5.0
clyent                             1.2.2
cmdstanpy                          0.4.0
colorama                           0.4.3
colorlover                         0.3.0
comtypes                           1.1.7
conda                              4.8.3
conda-build                        3.18.11
conda-package-handling             1.7.0
conda-verify                       3.4.2
confuse                            1.3.0
contextlib2                        0.6.0.post1
convertdate                        2.2.1
cryptography                       2.9.2
cycler                             0.10.0
Cython                             0.29.21
cytoolz                            0.10.1
dask                               2.22.0
datasist                           1.5.3
decorator                          4.4.2
defusedxml                         0.6.0
diff-match-patch                   20200713
distributed                        2.22.0
docutils                           0.16
entrypoints                        0.3
ephem                              3.7.7.1
et-xmlfile                         1.0.1
fastcache                          1.1.0
filelock                           3.0.12
flake8                             3.7.9
Flask                              1.1.2
fsspec                             0.7.4
future                             0.18.2
gast                               0.2.2
gensim                             3.8.3
gevent                             20.6.2
glob2                              0.7
gluoncv                            0.6.0
gmpy2                              2.0.8
google-auth                        1.17.2
google-auth-oauthlib               0.4.1
google-pasta                       0.2.0
GPUtil                             1.4.0
graphviz                           0.8.4
greenlet                           0.4.16
grpcio                             1.27.2
h5py                               2.10.0
HeapDict                           1.0.1
holidays                           0.10.2
html5lib                           1.1
htmlmin                            0.1.12
idna                               2.6
ImageHash                          4.1.0
imageio                            2.9.0
imagesize                          1.2.0
imbalanced-learn                   0.6.2
importlib-metadata                 1.7.0
iniconfig                          0.0.0
intervaltree                       3.0.2
ipykernel                          5.3.4
ipython                            7.16.1
ipython-genutils                   0.2.0
ipywidgets                         7.5.1
isort                              4.3.21
itsdangerous                       1.1.0
jdcal                              1.4.1
jedi                               0.15.2
Jinja2                             2.11.2
jmespath                           0.9.5
joblib                             0.16.0
json5                              0.9.5
jsonschema                         3.2.0
jupyter                            1.0.0
jupyter-client                     6.1.6
jupyter-console                    6.1.0
jupyter-core                       4.6.3
jupyter-kite                       1.0.0
jupyterlab                         2.2.2
jupyterlab-server                  1.2.0
Keras                              2.3.1
Keras-Applications                 1.0.8
Keras-Preprocessing                1.1.0
keras-resnet                       0.2.0
keyring                            21.2.1
kiwisolver                         1.2.0
korean-lunar-calendar              0.2.1
lazy-object-proxy                  1.4.3
libarchive-c                       2.9
llvmlite                           0.33.0+1.g022ab0f
locket                             0.2.0
LunarCalendar                      0.0.9
lxml                               4.5.2
Markdown                           3.1.1
MarkupSafe                         1.1.1
matplotlib                         3.3.0
mccabe                             0.6.1
menuinst                           1.4.16
missingno                          0.4.2
mistune                            0.8.4
mkl-fft                            1.1.0
mkl-random                         1.1.0
mkl-service                        2.3.0
mock                               4.0.2
more-itertools                     8.4.0
mpmath                             1.1.0
msgpack                            1.0.0
multipledispatch                   0.6.0
multitasking                       0.0.9
mxnet                              1.6.0
navigator-updater                  0.2.1
nbconvert                          5.6.1
nbformat                           5.0.7
networkx                           2.4
nltk                               3.5
nose                               1.3.7
notebook                           6.1.1
numba                              0.50.1
numexpr                            2.7.1
numpy                              1.19.1
numpydoc                           1.1.0
oauthlib                           3.1.0
olefile                            0.46
opencv-python                      4.2.0.32
openpyxl                           3.0.4
opt-einsum                         3.1.0
packaging                          20.4
pandas                             1.1.0
pandas-profiling                   2.8.0
pandocfilters                      1.4.2
paramiko                           2.7.1
parso                              0.5.2
partd                              1.1.0
path                               13.1.0
pathlib2                           2.3.5
pathtools                          0.1.2
patsy                              0.5.1
pep8                               1.7.1
pexpect                            4.8.0
phik                               0.10.0
pickleshare                        0.7.5
Pillow                             7.2.0
pip                                20.2.1
pkginfo                            1.5.0.1
plot-metric                        0.0.6
plotly                             4.8.2
pluggy                             0.13.1
ply                                3.11
portalocker                        1.6.0
prometheus-client                  0.8.0
prompt-toolkit                     3.0.5
protobuf                           3.12.3
psutil                             5.7.0
py                                 1.9.0
pyasn1                             0.4.8
pyasn1-modules                     0.2.7
pycodestyle                        2.5.0
pycosat                            0.6.3
pycparser                          2.20
pycrypto                           2.6.1
pycurl                             7.43.0.5
pydocstyle                         5.0.2
pydot                              1.4.1
pydotplus                          2.0.2
pyflakes                           2.1.1
Pygments                           2.6.1
PyJWT                              1.7.1
PyLg                               1.3.3
pylint                             2.5.3
PyMeeus                            0.3.7
PyNaCl                             1.4.0
pyodbc                             4.0.0-unsupported
pyOpenSSL                          19.1.0
pyparsing                          2.4.7
PyPDF2                             1.26.0
pyreadline                         2.1
pyrsistent                         0.16.0
PySocks                            1.7.1
pystan                             2.19.1.1
pytest                             6.0.1
python-dateutil                    2.8.1
python-jsonrpc-server              0.3.4
python-language-server             0.31.10
pytz                               2020.1
PyWavelets                         1.1.1
pywin32                            227
pywin32-ctypes                     0.2.0
pywinpty                           0.5.7
PyYAML                             5.3.1
pyzmq                              19.0.1
QDarkStyle                         2.8.1
QtAwesome                          0.7.2
qtconsole                          4.7.5
QtPy                               1.9.0
regex                              2020.7.14
requests                           2.24.0
requests-oauthlib                  1.3.0
retrying                           1.3.3
rope                               0.17.0
rsa                                4.0
Rtree                              0.9.4
ruamel-yaml                        0.15.87
s3transfer                         0.3.3
scikit-image                       0.16.2
scikit-learn                       0.22.1
scipy                              1.5.0
seaborn                            0.10.1
selenium                           3.141.0
Send2Trash                         1.5.0
setuptools                         49.2.1.post20200807
setuptools-git                     1.2
simplegeneric                      0.8.1
singledispatch                     3.4.0.3
six                                1.15.0
smart-open                         2.0.0
snowballstemmer                    2.0.0
sortedcollections                  1.2.1
sortedcontainers                   2.2.2
soupsieve                          2.0.1
Sphinx                             3.1.2
sphinxcontrib-applehelp            1.0.2
sphinxcontrib-devhelp              1.0.2
sphinxcontrib-htmlhelp             1.0.3
sphinxcontrib-jsmath               1.0.1
sphinxcontrib-qthelp               1.0.3
sphinxcontrib-serializinghtml      1.1.4
sphinxcontrib-websupport           1.2.3
spyder                             4.1.3
spyder-kernels                     1.9.2
SQLAlchemy                         1.3.18
squarify                           0.4.3
statsmodels                        0.11.1
stem                               1.8.0
sympy                              1.6.1
tables                             3.6.1
tabulate                           0.8.3
tangled-up-in-unicode              0.0.6
tblib                              1.6.0
tbselenium                         0.4.2
tensorboard                        2.2.1
tensorboard-plugin-wit             1.6.0
tensorflow                         2.1.0
tensorflow-estimator               2.1.0
termcolor                          1.1.0
terminado                          0.8.3
testpath                           0.4.4
textblob                           0.15.3
threadpoolctl                      2.1.0
toml                               0.10.1
toolz                              0.10.0
torch                              1.4.0
torchvision                        0.5.0
tornado                            6.0.4
tqdm                               4.48.2
traitlets                          4.3.3
typed-ast                          1.4.1
typing-extensions                  3.7.4.2
ujson                              1.35
unicodecsv                         0.14.1
urllib3                            1.22
vaderSentiment                     3.3.1
visions                            0.4.4
watchdog                           0.10.3
wcwidth                            0.2.5
webencodings                       0.5.1
Werkzeug                           0.15.4
wheel                              0.34.2
widgetsnbextension                 3.5.1
win-inet-pton                      1.1.0
win-unicode-console                0.5
wincertstore                       0.2
wrapt                              1.11.2
xgboost                            1.0.2
xlrd                               1.2.0
XlsxWriter                         1.2.9
xlwings                            0.19.5
xlwt                               1.3.0
xmltodict                          0.12.0
yapf                               0.30.0
yfinance                           0.1.54
zict                               2.0.0
zipp                               3.1.0
zope.event                         4.4
zope.interface                     4.7.1

The output of jupyter labextension list:
JupyterLab v2.2.2
Known labextensions:
   app dir: C:\ProgramData\Anaconda3\share\jupyter\lab
        @kiteco/jupyterlab-kite v1.0.0 enabled  ok

![kite](https://user-images.githubusercontent.com/30699597/89711279-e64fa800-d9a6-11ea-991a-c86bf64534c1.PNG)
![jupyter](https://user-images.githubusercontent.com/30699597/89711296-f49dc400-d9a6-11ea-9328-76bba62e27a0.PNG)

**Comment by its-dhung** [comment]
A couple of questions:

* What is the full path of the notebook you are working in?
* Do you see the Kite icon in the bottom left corner of your browser?
* What gets printed to your browser's JavaScript console as you type code into the notebook?

**Comment by swankyshahir** [comment]
it is working properly... just that i was starting jupyter notebook instead of jupyter lab....
